screens/product listing page/product.css

```diff
@@ -0,0 +1,98 @@
+@import url("/home.css");
+
+.grid-container {
+  display: grid;
+  grid-template-areas:
+    "navbar navbar"
+    "filter-list products-category-box"
+    "footer footer";
+  column-gap: 1.5rem;
+  grid-row-gap: 1rem;
+}
+
+.nav {
+  grid-area: navbar;
+  border-bottom: 1px solid var(--border-shade);
+  width: 100%;
+  position: sticky;
+  z-index: 1;
+  top: 0;
+}
+/* product list  box*/
+.products-category-box {
+  grid-area: products-category-box;
+  overflow-x: hidden;
+}
+/* filter list */
+.filter-list {
+  grid-area: filter-list;
+  width: max-content;
+  border-right: 1px solid var(--theme-shade-3);
+  padding: 1rem;
+  height: calc(100vh - 6rem);
+  overflow: scroll;
+  position: sticky;
+  top: 10%;
+  left: 1%;
+  border-top: 1px solid var(--border-shade);
+}
+/* footer */
+.footer {
+  grid-area: footer;
+  height: 15vh;
+  position: relative;
+  z-index: 1;
+}
+/* different products */
+.products-list {
+  flex-wrap: wrap;
+  justify-content: space-between;
+  gap: 1rem;
+  margin-bottom: 5rem;
+}
+input[type="radio"] {
+  accent-color: var(--theme-color);
+}
+
+/* overriding styles */
+.title {
+  margin: 0rem auto;
+  margin-bottom: 2rem;
+  line-height: 3rem;
+}
+.card-img {
+  height: 20vh;
+  width: fit-content;
+  margin-top: 1rem;
+}
+.bd-bottom {
+  border-bottom: 1px solid var(--theme-shade-3);
+}
+
+.flex-column {
+  justify-content: space-between;
+  padding: 1rem;
+  gap: 0.5rem;
```

Comment:
Suggestion: you can add this in utils and use it wherever needed.
Also you can use pixels for values less than 1 rem.

Reply:
I have added other majorly used flex styling already in my utils, it's specifically used for here only

Reply:
Yes ,I will do it